backend/utils/deployShop.js

```diff
@@ -44,6 +47,70 @@ function urlToMultiaddr(v, opts) {
   )}${url.pathname}`
 }
 
+/**
+ * Trigger AutoSSL to generate cert
+ *
+ * @param url {T} URL object or string to connect to
+ * @returns {boolean} if it was successful
+ */
+async function triggerAutoSSL(url, autoSSLHost) {
+  let success = false
+  let attempts = 1
+
+  url = url instanceof URL ? url : new URL(url)
+  const hostname = url.hostname
+  url.hostname = autoSSLHost.includes('//')
+    ? autoSSLHost.split('//').slice(-1)
+    : autoSSLHost
+
+  log.debug(`Making request to ${url.hostname} with header (Host: ${hostname})`)
+
+  for (;;) {
+    try {
+      await getHTTPS({
+        host: url.hostname,
+        path: url.pathname,
+        port: url.port || 443,
+        servername: hostname,
+        headers: {
+          host: hostname
+        }
+      })
+      success = true
+      break
+    } catch (err) {
+      log.info(
+        `Error when attempting to trigger AutoSSL on attempt ${attempts}:`,
+        err
+      )
+
+      if (attempts <= AUTOSSL_MAX_ATTEMPTS) {
+        // sleep with backoff
+        await (async () => {
+          log.debug(
+            `AutoSSL trigger backing off by ${attempts * AUTOSSL_BACKOFF}ms`
+          )
+          return new Promise((resolve) =>
+            setTimeout(resolve, attempts * AUTOSSL_BACKOFF)
```

Comment:
Worst case scenario, that's a wait of 30s + 2 * 30s + 3 * 30s + 4 * 30s + 5 * 30s = 7.5min which seems a bit long?
Should AUTOSSL_BACKOFF get lowered a bit?

Reply:
The backoff was chosen specifically because it was > 5min.  This is intentional.